client-libs/python/openpipe/__init__.py

```diff
@@ -2,3 +2,5 @@
 from openai import *
 from .openai_sync_wrapper import OpenAIWrapper as OpenAI
 from .openai_async_wrapper import AsyncOpenAIWrapper as AsyncOpenAI
+
+from .api_client_fern.client import OpenPipeApi, AsyncOpenPipeApi
```

Comment:
In another PR we should rename `api_client_fern` to `api_client`. I don't want to do that in this change because it would be confusing to track what files were added/deleted.